.circleci/config.yml

```diff
@@ -67,6 +68,7 @@ commands:
       - run:
           name: Switch dof-dss packages to HEAD on development branch
           command: |
+            cd project
```

Comment:
A very tiny detail, but you're better off referencing `$PROJECT_ROOT` here instead of the literal directory name, as this value can change depending on your docker image. You can see above when moving from circleci's image that the username differs and it's easier to change it in one variable than piece by piece the full config file.

Reply:
In fact, this still works as it's the username that differs, but other steps would reference the root path with the variable so it's handy for consistency if nothing else :)